### PR TITLE
[14.0][ADD] partner_contact_company_propagation module

### PR DESCRIPTION
**Pull request by Kev-Roche**
Small module allowing to propagate company_id between parents and children contacts.

### CODE REVIEW

**Comment by Kev-Roche** [comment]
It is a small module but very helpfull, @pedrobaeza, @bealdav, @sebastienbeau what do you think about it?

**Comment by pedrobaeza** [comment]
The name is not suitable, as this has nothing to do with intercompany itself. It should be named `partner_contact_company_propagation` or similar (for being inline with https://github.com/OCA/sale-workflow/tree/13.0/partner_contact_sale_info_propagation).

File: partner_contact_company_propagation/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Akretion (https://www.akretion.com).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Partner Contact Company Propagation",
+    "summary": "Share company name between Partner's contacts",
```
Comment (pedrobaeza) [inline partner_contact_company_propagation/__manifest__.py]:
```suggestion
    "summary": "Propagate company info to children contacts",
```

**Review by pedrobaeza — APPROVED** [review]

**Review by ajaniszewska-dev — APPROVED** [review]

File: partner_contact_company_propagation/__manifest__.py (hunk 1 above)
Comment (Kev-Roche) [reply partner_contact_company_propagation/__manifest__.py]: Indeed, it's clearer that way. Thank you

**Pull request merged**